main.tf

```diff
@@ -4,6 +4,19 @@
 
 resource "aws_security_group" "postgresql" {
   vpc_id = "${var.vpc_id}"
+  ingress {
```

Comment:
I should have looked at this closer before. I don't think we want to make the assumption that module consumers will want their database open to the VPC. We got away from those assumptions in https://github.com/azavea/terraform-aws-postgresql-rds/pull/14.